src/govuk/tools/_compatibility.scss

```
@@ -24,13 +38,13 @@
 ///   this product
 /// @throw Errors if product name is not recognised
 /// @access public
+/// @deprecated Will be removed in v5.0 with the rest of the compatibility mode
+/// suite of tools and settings
 
 @mixin govuk-compatibility($product) {
-  @if map-has-key($_govuk-compatibility, $product) {
-    @if map-get($_govuk-compatibility, $product) == true {
-      @content;
-    }
-  } @else {
-    @error "Non existent product '#{$product}'";
+  @warn "govuk-compatibility is deprecated. " +
+    "From version 5.0, Govuk-frontend will not support legacy codebases";
```

I think 'legacy codebases' is too ambiguous here. Can we just say we're getting rid of compatibility mode?

```suggestion
    "From version 5.0, GOV.UK Frontend will not support compatibility mode";
```